.circleci/config.yml

```diff
@@ -161,49 +203,79 @@ jobs:
       - store_artifacts:
           path: test-results
 
-  unpackaged-sanity:
+
+  linux-tarball-test:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/repo
+
+      # We'll want to use the LTS version of node
+      - update_node:
+          os: linux
+
+      # Set up the plugin
+      - run:
+          name: "Install SFDX and tarball"
+          command: |
+            npm install -g sfdx-cli
+            # We need to determine the name of the tarball
+            SFDC_TARBALL=$(ls | grep salesforce*.tgz)
```

Comment:
It would be good to make this more explicit since we know the exact format of the file name. Something like `salesforce-sfdx-scanner-[0-9]*\\.[0-9]*\\.[0-9]*\\.tgz`